<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetFanOutForm</class>
 <widget class="QWidget" name="WidgetFanOutForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lTitle">
     <property name="text">
      <string>Fan Output</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="wChart" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Fan Stall Detection and Forwarding</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Include fans for stalled fan detection:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="cbDetectFan0">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="cbDetectFan1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="cbDetectFan2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="cbDetectFan3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QCheckBox" name="cbDetectFan4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QCheckBox" name="cbDetectFan5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Fans detected as stalled</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="FanStalledWidget" name="fsFan0">
          <property name="toolTip">
           <string>FanStalledWidget</string>
          </property>
          <property name="whatsThis">
           <string>FanStalledWidget.</string>
          </property>
          <property name="fanName">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="FanStalledWidget" name="fsFan1">
          <property name="toolTip">
           <string>FanStalledWidget</string>
          </property>
          <property name="whatsThis">
           <string>FanStalledWidget.</string>
          </property>
          <property name="fanName">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="FanStalledWidget" name="fsFan2">
          <property name="toolTip">
           <string>FanStalledWidget</string>
          </property>
          <property name="whatsThis">
           <string>FanStalledWidget.</string>
          </property>
          <property name="fanName">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="FanStalledWidget" name="fsFan3">
          <property name="toolTip">
           <string>FanStalledWidget</string>
          </property>
          <property name="whatsThis">
           <string>FanStalledWidget.</string>
          </property>
          <property name="fanName">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="FanStalledWidget" name="fsFan4">
          <property name="toolTip">
           <string>FanStalledWidget</string>
          </property>
          <property name="whatsThis">
           <string>FanStalledWidget.</string>
          </property>
          <property name="fanName">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="FanStalledWidget" name="fsFan5">
          <property name="toolTip">
           <string>FanStalledWidget</string>
          </property>
          <property name="whatsThis">
           <string>FanStalledWidget.</string>
          </property>
          <property name="fanName">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="FanStalledWidget" name="fsGlobal">
        <property name="toolTip">
         <string>FanStalledWidget</string>
        </property>
        <property name="whatsThis">
         <string>FanStalledWidget.</string>
        </property>
        <property name="fanName">
         <string>Stalled</string>
        </property>
        <property name="fanStalled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Fan Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Fan</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Duty cycle:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="spRpmOut">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string>rpm</string>
        </property>
        <property name="maximum">
         <number>30000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spDutyCycleIn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FanStalledWidget</class>
   <extends>QWidget</extends>
   <header>fanstalledwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
